### PR TITLE
Switch accounts

### PR DESCRIPTION
**Pull request by deanrobertcook**
#### APK
[Download build #9011](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9011/artifact/build/artifact/wire-dev-PR926-9011.apk)
[Download build #9013](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9013/artifact/build/artifact/wire-dev-PR926-9013.apk)
[Download build #9025](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9025/artifact/build/artifact/wire-dev-PR926-9025.apk)
[Download build #9044](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9044/artifact/build/artifact/wire-dev-PR926-9044.apk)
[Download build #9045](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9045/artifact/build/artifact/wire-dev-PR926-9045.apk)
[Download build #9050](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9050/artifact/build/artifact/wire-dev-PR926-9050.apk)
[Download build #9054](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9054/artifact/build/artifact/wire-dev-PR926-9054.apk)

### CODE REVIEW

**Comment by deanrobertcook** [comment]
build again

**Comment by deanrobertcook** [comment]
build again

**Comment by deanrobertcook** [comment]
will get QA review after all preferences are done

File: app/src/main/scala/com/waz/zclient/preferences/PreferencesActivity.scala [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+  override def onCreate(@Nullable savedInstanceState: Bundle) = {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_settings)
+    setSupportActionBar(toolbar)
+    titleSwitcher //initialise title switcher
+
+    if (LayoutSpec.isPhone(this)) ViewUtils.lockScreenOrientation(Configuration.ORIENTATION_PORTRAIT, this)
+    if (savedInstanceState == null) getSupportFragmentManager.beginTransaction.add(R.id.content, RootPreferences.newInstance(null, getIntent.getExtras), RootPreferences.TAG).commit
+
+//    currentAccountPref.signal.onChanged {
```
Comment (makingthematrix) [inline app/src/main/scala/com/waz/zclient/preferences/PreferencesActivity.scala]: Please remove unused code

**Comment by makingthematrix** [comment]
looks good

**Comment by deanrobertcook** [comment]
build again

**Pull request merged**